src/videojs.markers.js

```diff
@@ -393,13 +392,16 @@ function registerVideoJsMarkersPlugin(options) {
     }
     onTimeUpdate();
     player.on("timeupdate", onTimeUpdate);
-    player.off("loadedmetadata");
+    // player.off("loadedmetadata");
   }
 
   // setup the plugin after we loaded video's meta data
-  player.on("loadedmetadata", function() {
+  // player.on("loadedmetadata", function() {
+  //   initialize();
+  // });
+  player.ready(function(){
```

Comment:
This is what needs to be done to isolate the off loadedmetadata event listener to any other loadedmetadata listeners:
function initialize() {
        if (setting.markerTip.display) {
          initializeMarkerTip();
        }

        // remove existing markers if already initialized
        player.markers.removeAll();
        addMarkers(setting.markers);

        onTimeUpdate();
        player.on("timeupdate", onTimeUpdate);
        player.off("loadedmetadata", initialize);
      }

      // setup the plugin after we loaded video's meta data
      player.on("loadedmetadata", initialize);